### ISSUE TITLE
Could I specify my own models or only the mentioned models only.

### ISSUE THREAD

**Issue by alqurri77** [tomgoldstein/loss-landscape#46]
Hi

I wonder if this implementation is for the models that is mentioned or if I can specify my own model.

**Comment by tirtho109** [comment]
I have the same question. It would be nice to get some feedback.

**Comment by alqurri77** [comment]
I found this code that allows you to use your model: https://github.com/marcellodebernardi/loss-landscapes

**Comment by sndnyang** [comment]
of course, you can use your custom model(s) to output value(s) and plot landscape.
